test/gomega/transforms/curl.go

```diff
@@ -28,6 +28,7 @@ const (
 func WithCurlHttpResponse(curlResponse string) *http.Response {
 	headers := make(http.Header)
 	statusCode := 0
+	protocol := ""
```

Comment:
Does this work (and some previous work) render the comment on this function out of date? (that references we handle http/1.1 headers).

Should we default the protocol to some value, or is there a benefit in leaving the "emtpy string" as a sentinel that we could not parse the protocol?

Reply:
Thanks for pointing this out, updated the comments
I think that an empty string is better to indicate that the protocol was not parsed